doc/ch-image_desc.rst

```diff
@@ -534,6 +555,50 @@ Same, except place the image in :code:`/tmp/buster`::
    bin   dev  home  lib64  mnt  proc  run   srv  tmp  var
    boot  etc  lib   media  opt  root  sbin  sys  usr
 
+Print list of targetable architecturess for the Debian Buster image.::
+
+   $ ch-image pull --list-arch debian:buster
+   fat manifest: using existing
+   available platforms:
+   amd64
+   arm/v5
+   arm/v7
+   arm64/v8
+   386
+   mips64le
+   ppc64le
+   s390x
+
+Specify the aarch64 version of Debian Stretch using :code:`--arch`.::
```

Comment:
```suggestion
Specify the aarch64 version of Debian Stretch using :code:`--arch`::
```

Reply:
The other examples in this file have a period. Should I remove them as well?